### ISSUE TITLE
Creating a dashboard including Ontime

### ISSUE THREAD

**Issue by sagerian** [cpvalente/ontime#356]
I'm preparing for a job that is coming up. The job is a 6 week build for a a large touring exhibition.

I want to create a web dashboard to display pertinent information around site during the build.

I want to use Ontime to handle a clock and to display timers that will keep track of start of day, breaks, lunch, etc.

I have a spreadsheet that is a list of tasks/jobs that must be completed before the job is done. I am not opposed to moving this to a different database.

I want to display these tasks on the dashboard and filter them to show tasks with a nearing due date or a high priority etc. I've been looking at Datatables as an option for this.

Ideally the dashboard would display nice to have information such as the weather, important notifications, local pub specials, etc.

I've had a play with the [Ontime Viewer Template](https://github.com/cpvalente/ontime-viewer-template) and I can get it to load but I can't see any values. I suspect that this is because the API has changed since this was created.

I'm a bit of a novice here but I've made some progress. I've successfully cloned the ontime repo and can compile/run locally. I've been playing with [DataTables](https://datatables.net/) and can run the [Zero Config](https://datatables.net/examples/basic_init/zero_configuration.html) example locally.

Can anyone suggest where I should be looking next? or can anyone offer any help/tips.

**Comment by cpvalente** [comment]
Hi @sagerian , thank you for reaching out

Which version of ontime are you using? Keep in mind that v2 (currently in beta) has broken changes and a new template:
- template for v1: https://github.com/cpvalente/ontime-viewer-template
- template for v2: https://github.com/cpvalente/ontime-viewer-template-v2

to our best knowledge, both templates are operational. If this is not the case, please follow up with any errors and we can try and help

As for data tables, I am afraid I cant help here, but I can help with the API setup at least

Here would be my suggestions:
- use ontime v2 and its template to start your project,
- make a fork (or copy-paste) from the v2 template and use this as a start for your project
- if you run into any trouble with the template, please open issues in the repo

Once you have a fork with a minimal setup of what you are looking for, I am happy to take a look and help with the ontime bit

Hope this helps
